src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -29,6 +30,52 @@ class TimeSeries(Table):
     # ------------------------------------------------------------------------------------------------------------------
     # Creation
     # ------------------------------------------------------------------------------------------------------------------
+
+    @staticmethod
+    def timeseries_from_csv_file(
+        path: str | Path,
+        target_name: str,
+        time_name: str,
+        feature_names: list[str] | None = None,
+    ) -> TimeSeries:
+        """
+        Read data from a CSV file into a table.
+
+        Parameters
+        ----------
+        path :
+            The path to the CSV file.
+
+        target_name:
+            The name of the target column
+
+        time_name :
+            The name of the time column
+
+        feature_names:
+            The name(s) of the column(s)
+
+
+        Returns
+        -------
+        table :
+            The time series created from the CSV file.
+
+        Raises
+        ------
+        FileNotFoundError
+            If the specified file does not exist.
+        WrongFileExtensionError
+            If the file is not a csv file.
```

Comment:
Add the errors from the `TimeSeries._from_table` method

Reply:
added